package.json

```diff
@@ -14,9 +14,10 @@
     "release": "./release.sh",
     "publish-release": "./publish.sh",
     "precommit": "lint-staged",
-    "test": "yarn test:unit && yarn test:e2e",
+    "test": "yarn test:unit && yarn test:e2e && yarn test:e2e:healthcare",
```

Comment:
Does healthcare use a relative path to core resources or the official released version?
If it's the latter, I have a question about coupling healthcare tests with core tests...

Reply:
ds-healthcare-gov references the core package through its package name. Lerna hoists the packages up to a root node_modules, and the package is symlinked there